sycl/unittests/CMakeLists.txt

```diff
@@ -1,12 +1,22 @@
-add_custom_target(PiUnitTests)
-set_target_properties(PiUnitTests PROPERTIES FOLDER "Tests")
+add_custom_target(SYCLUnitTests)
+set_target_properties(SYCLUnitTests PROPERTIES FOLDER "SYCL tests")
```

Comment:
Aligns testing and unittests with other LLVM projects CMake usage, e.g., Clang

Reply:
Idem.

Reply:
Other LLVM projects I checked do not comment on where they copied and adapted from other LLVM projects though it is clear that they are aligned.